src/common/example/index.ts

```diff
@@ -1,8 +1,31 @@
 import { WidgetBase } from '@dojo/widget-core/WidgetBase';
-import { WidgetProperties } from '@dojo/widget-core/interfaces';
 import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
 import Select from '../../select/Select';
 import { v, w } from '@dojo/widget-core/d';
+import { Registry } from '@dojo/widget-core/Registry';
+import dojoTheme from '../../themes/dojo/theme';
+
+import accordionpane from './../../accordionpane/example/index';
```

Comment:
I wish we had a convenience index module so this isn't so verbose. We could technically have one that just isn't built. This is rhetorical, just thinking out loud.